charts/ctlog/templates/_helpers.tpl

```diff
@@ -98,10 +98,18 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 Selector labels
 */}}
 {{- define "ctlog.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "ctlog.name" . }}
+app.kubernetes.io/name: {{ .Values.name | quote }}
 app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/component: {{ .Values.name | quote }}
```

Comment:
this should probably switch to using `{{ include "ctlog.name" . }}`
same in `matchLabels`